docs/api-and-integration/octopus.server.exe-command-line/path.md

```diff
@@ -13,6 +13,11 @@ Usage: octopus.server path [<options>]
 Where [<options>] is any of:
 
       --instance=VALUE       Name of the instance to use
+      --cacheDirectory=VALUE Directory to use for temporary files and cachin-
+                               g, e.g. downloaded packages. This data in this
```

Comment:
Minor note for the next update, "This data in this directory" while technically accurate, would read better imo as "The data in this directory".

Reply:
Tweaked, @wordlee - https://github.com/OctopusDeploy/OctopusDeploy/commit/2197ca98c7c98bbfc289a7811a3e96849c20462b

Reply:
👍